otelcollector/deploy/addon-chart/ccp-metrics-plugin/templates/ama-metrics-role.yaml

```diff
@@ -0,0 +1,27 @@
+# Source: ama-metrics-cpp/templates/ama-metrics-role.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ama-metrics-ccp-role
+  namespace: {{ .Values.global.commonGlobals.Customer.Namespace }}
+rules:
+  - apiGroups: [""]
+    resources:
+      [
+        "pods",
+        "services",

```

Comment:
i dont think i see any services based discovery for ccp ? only pods & endpoints ?

Reply:
Hey @sgeannina, adding you in as you had made this file. Is there any plan on adding service based discovery for ccp?

Reply:
we don't have too many services in the control plane, pods and endpoints should be enough